frontend/rolecall/src/app/api/cast_api.service.ts

```diff
@@ -192,95 +199,73 @@ export class CastApi {
     return this.mockBackend.requestOneCast(uuid);
   };
 
-  /** Hits backend with create/edit cast POST request */
-  async requestCastSet(cast: Cast): Promise<HttpResponse<any>> {
-    if (environment.mockBackend) {
-      return this.mockBackend.requestCastSet(cast);
-    }
-    // Check if we have record of the cast and patch if we do
-    if (this.hasCast(cast.uuid)) {
-      // Do patch
-      let header = await this.headerUtil.generateHeader();
-      return this.http.delete(environment.backendURL + 'api/cast?castid=' + cast.uuid, {
+/** Hits backend with create/edit cast POST request */
```

Comment:
Was the indentation amount changed here?

Reply:
I broke out completely duplicated code it the two branches ('Do patch' and 'Do post'). The remaining code indentation level may have changed as a result.

Reply:
That's ok, but could you please fix the indent on this `requestCastSet()` function? Currently diff is showing every line as changed when it actually was not.

![image](https://user-images.githubusercontent.com/6393085/95147479-b5d18380-0735-11eb-9aaa-a47c1eb2b409.png)